### PR TITLE
add test data set link; move orphans.fq.gz to orphans.qc.fq.gz

### PR DESCRIPTION
**Pull request by ctb**
ping @ljcohen @standage

### CODE REVIEW

**Comment by ctb** [comment]
closes #5 at least for AWS install.

File: 1-quality.rst [hunk 1]
```diff
@@ -9,11 +9,16 @@ Make sure you've got the PROJECT location defined, and your data is there:
 
    set -u
    printf "\nMy raw data is in $PROJECT/data/, and consists of $(ls -1 ${PROJECT}/data/*.fastq.gz | wc -l) files\n\n"
+   set +u
```
Comment (standage) [inline 1-quality.rst:12]: Any reason we want to allow unbound variables? Might be better to proceed in "safe mode". \*shrug\*

File: 1-quality.rst [hunk 2]
```diff
@@ -127,14 +132,14 @@ Run:
...
+        gzip -9c s1_se s2_se >> orphans.qc.fq.gz
         rm -f s1_se s2_se
    done
 
 
 The paired sequences output by this set of commands will be in the
-files ending in ``qc.fq.gz``, with any orphaned sequences all together
-in ``orphans.fq.gz``.
+files ending in ``qc.qc.fq.gz``, with any orphaned sequences all together
+in ``orphans.qc.fq.gz``.
```
Comment (standage) [inline 1-quality.rst:142]: Is this correct?

**Review by standage — COMMENTED** [review]
LGTM, but curious about responses to my comments.

File: 1-quality.rst (hunk 1 above)
Comment (standage) [reply 1-quality.rst:12]: `set -u` disables tab auto-complete, so we indeed want to use it sparingly.

File: 1-quality.rst (hunk 2 above)
Comment (standage) [reply 1-quality.rst:142]: To be clear: is `qc.qc` correct?

Comment (ctb) [reply 1-quality.rst:142]: Nope! fixed.

**Review by standage — APPROVED** [review]
Great

**Pull request merged**